src/main/java/duke/Task.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+public abstract class Task {
+
+    private String description;
+    private boolean isDone;
+
+    public Task(String desc) {
+        this.description = desc;
+        this.isDone = false;
+    }
+
+    public String getStatus() {
+        return isDone ? "V" : "X";
+    }
+
+    public void setStatus(boolean status) {
+        this.isDone = status;
+    }
+
+    public String getDescription() {
```

Comment:
Good use of private field + getters and setters which were not required by the iP tasks